### ISSUE TITLE
RevG with boot loop

### ISSUE THREAD

**Issue by ElDominio** [rusefi/alphax-2chan#112]
I have a revGboot looping if powered from 12V, but works fine if first powered by 5v. 

I have tride Full DFU Erase and reloading firmware, but not improvement

**Comment by rusefillc** [comment]
@ElDominio I see a statement. I approve this statement.

What is the question? Are you looking to check if all 2chan rev G do the same? Something else?

**Comment by ElDominio** [comment]
Yes, I would like if others do the same, and if this is a software or hardware problem

Two RevFs I have tested in the same car boot properly, but VR does not work on them for reasons unknown

**Comment by ElDominio** [comment]
I found an old revG with 2023 firmware, and it was booting up fine. Updated to latest and revG does not boot loops and RARELY finishes booting. RevF boots fine

**Comment by rusefillc** [comment]
@ElDominio I have rev G and I am able to reproduce the issue. I have to confess that I was totally NOT expecting this to be a firmware issue.

@ElDominio I need help. Can you please run a binary search between official releases and/or daily builds and attempt to isolate the issue?

**Comment by ElDominio** [comment]
I've already downgraded to Dec2023 and still boot loop :(

**Comment by ElDominio** [comment]
Feb2023 is ok!

https://github.com/rusefi/rusefi/releases?page=57
https://github.com/rusefi/rusefi/releases/tag/2023-03-09
![image](https://github.com/user-attachments/assets/a375ea52-4fe1-4ea8-825c-b8c4c5401bba)

**Comment by ElDominio** [comment]
the following are broken:
2023-04-19
June2023 is broken

**Comment by rusefillc** [comment]
2023-03-09 good

https://github.com/rusefi/rusefi/releases/tag/2023-04-03 OK

https://github.com/rusefi/rusefi/releases/tag/2023-04-10 OK

https://github.com/rusefi/rusefi/releases/tag/2023-04-16 OK

https://github.com/rusefi/rusefi/releases/tag/2023-04-17 OK

https://github.com/rusefi/rusefi/releases/tag/2023-04-18 bad

2023-04-19 bad

**Comment by rusefillc** [comment]
https://github.com/rusefi/rusefi/tree/2chan-revG

https://github.com/rusefi/rusefi/tree/branch-2023-04-20-stamp

**Comment by rusefillc** [comment]
17 in file name is 16 in date_stamp?

firmwareInfo="rusEFI+v20230416%4054879" nPages="1" signature="rusEFI master.2023.04.17.alphax-2chan.3461928810"

https://github.com/rusefi/rusefi/releases/tag/2023-04-17 
![image](https://github.com/user-attachments/assets/4b5d8ed6-3109-491b-9d5b-ab83f6cd3c5c)

**Comment by rusefillc** [comment]
smoking gun https://github.com/rusefi/rusefi/commit/b135fecfe9f61562ff655c86f236f5079e949457

**Comment by rusefillc** [comment]
![image](https://github.com/user-attachments/assets/4f8b86ef-b2fd-4f14-862e-2be044456820)

**Comment by rusefillc** [comment]
rev F board is happy with latest firmware and 12v power supply

```
2024-10-06_11_23_51_236: EngineState: hellenBoardId=115
2024-10-06_11_24_11_341: EngineState: pin PE10 ((null)): EN Mode GP out AF0 Push-Pull No pull Speed Low IN 1 OUT 1
```

**Comment by rusefillc** [comment]
rev G is NOT happy with latest fw if started from 12v
```
2024-10-06_11_24_50_336: EngineState: hellenBoardId=210
2024-10-06_11_25_24_413: EngineState: pin PE10 ((null)): EN Mode GP out AF0 Push-Pull No pull Speed Low IN 1 OUT 1
```

**Comment by rusefillc** [comment]
@Qwerty-OFF https://github.com/rusefi/alphax-2chan/issues/100

**Comment by rusefillc** [comment]
@ElDominio shit that's https://github.com/andreika-git/hellen-one/issues/353

please remove two capacitors from rev G boards to have them boot. So far our software attempts to fix have failed

rev F i have it happy
![WhatsApp Image 2024-10-06 at 11 42 33_2875ec82](https://github.com/user-attachments/assets/29617c4b-ffe9-4cf6-a1d6-6b1c48bc9d85)


rev G is happy after removal
![WhatsApp Image 2024-10-06 at 11 43 34_b07d5372](https://github.com/user-attachments/assets/8ef6d4b8-f4dd-4e66-9546-411a8430355a)

**Comment by ElDominio** [comment]
fix confirmed!!!b

**Comment by ElDominio** [comment]
This is fixed, closing
